### ISSUE TITLE
STM32L452 not found in database.

### ISSUE THREAD

**Issue by harperjohn** [fpoussin/QStlink2#49]
I see that the following (see below) entry is in the xml file, yet my STM32L452 is "not found in database"...

Does this mean that a different (or more specific) entry needs to be added to the xml file?

I would be glad to make the change... are there (detailed) instructions on how to add *and* test the entry?

Thanks!!

from the xml file:
device type="STM32L4xx" coretype="CM4F"

**Comment by fpoussin** [comment]
Yes, it needs to match the correct coreID/chipID from the datasheets.
Maybe you have a newer revision with different IDs.
